UIKit/Location Controller/TWTLocationController.m

```diff
@@ -0,0 +1,189 @@
+//
+//  TWTLocationController.m
+//  Toast
+//
+//  Created by Steve Foster on 2/3/15.
+//  Copyright (c) 2015 Two Toasters, LLC. All rights reserved.
+//
+
+#import "TWTLocationController.h"
+
+@import CoreLocation;
+
+static NSTimeInterval const kLocationTimeoutInterval = 60;
+NSString *const kLocationControllerErrorDomain = @"locationcontroller";
```

Comment:
Error domains should be made public, else they’re not really useful. I’d call it `TWTLocationControllerErrorDomain` and make the string the same (“TWTLocationControllerErrorDomain”).